mqtt/mqtt-broker-tests-util/src/server.rs

```diff
@@ -101,6 +101,10 @@ impl DummyAuthenticator {
     pub fn anonymous() -> Self {
         Self(AuthId::Anonymous)
     }
+
+    pub fn id(id: &str) -> Self {
```

Comment:
nit: better naming for constructor func `with_id()`, can be confused with id property getter.